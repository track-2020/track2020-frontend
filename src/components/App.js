import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from 'react-router-dom';
import Login from '../pages/Login';
// import Signup from '../pages/Signup';

export default function App() {
  return (
    <Router>
      <Switch>
        <Route exact path="/" component={Login} />
        {/* <Route path="/signup" component={Signup} />    */}
      </Switch> 
    </Router>
  );
}
