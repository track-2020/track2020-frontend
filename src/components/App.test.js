import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App component', () => {
  it('renders the App component', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
