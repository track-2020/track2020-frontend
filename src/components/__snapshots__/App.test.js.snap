// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the App component 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/signup"
    />
  </Switch>
</BrowserRouter>
`;
